site2/website/versioned_sidebars/version-2.7.1-sidebars.json

```diff
@@ -150,7 +150,6 @@
     "Reference": [
       "version-2.7.1-reference-terminology",
       "version-2.7.1-reference-cli-tools",
-      "version-2.7.1-pulsar-admin",
```

Comment:
This will remove the pulsar-admin cli page from the website, right? I think it would be really valuable to leave this page in place and to put a banner at the top to indicate the page is deprecated and that there is a new/updated pulsar admin cli website. As a user, I'd be surprised if this page would just suddenly disappear because I didn't know it was out of date or that there was a new/different place to look for the pulsar admin cli documentation.